src/main/java/com/cloudbees/jenkins/GitHubRepositoryNameContributor.java

```diff
@@ -46,7 +76,7 @@
 
 
     static abstract class AbstractFromSCMImpl extends GitHubRepositoryNameContributor {
```

Comment:
This would be a refactor that will make the merge after #57 even harder, I'd prefer to keep it as is now.